docs/observability/reliability-management-slo/dashboards.md

```diff
@@ -62,12 +62,23 @@ Each dashboard contains the following information:<br/><img src={useBaseUrl('img
 </table>
 
 
-## Setting Granularity for SLO Time Ranges
+## Setting SLO Data Granularity
+
+### Time Ranges
 
 To modify the time range, select and drag across dates to zoom in further. This can be useful if you want to zoom in for granular details, especially for charts with larger compliance periods:<br/><img src={useBaseUrl('img/observability/2022-08-02_14-29-19.gif')} alt="Reliability Management SLO SLI" />
```

Comment:
```suggestion
To modify the time range, select and drag across dates to zoom in further. This can be useful if you want to zoom in for granular details, especially for charts with larger compliance periods.<br/><img src={useBaseUrl('img/observability/2022-08-02_14-29-19.gif')} alt="Reliability Management SLO SLI" />
```

Reply:
Removed  `:`